content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -275,33 +275,79 @@ node {
 }
 ----
 
-=== Defining steps
 
+=== Defining global variables
+
+Internally, scripts in the `vars` directory are instantiated as a singleton
+on-demand, when used first.  So it is possible to define more methods,
```

Comment:
"So it is possible" reads like a sentence fragment. Should it be something like "more methods or properties?"